app-crypt/libsecret/libsecret-0.20.4.ebuild

```diff
@@ -71,16 +71,12 @@ src_prepare() {
 		-i libsecret/secret-enum-types.h.template || die
 }
 
-meson_multilib_native_use() {
-	multilib_native_usex "$1" "-D${2-$1}=true" "-D${2-$1}=false"
-}
-
 multilib_src_configure() {
 	local emesonargs=(
 		$(meson_use crypt gcrypt)
 
 		# Don't build docs multiple times
-		-Dmanpage=$(multilib_is_native_abi && echo true || echo false)
+		$(meson_native_true manpage)
 		$(meson_multilib_native_use gtk-doc gtk_doc)
```

Comment:
But you've renamed the function, right?

Reply:
Damn. Not sure how that slipped through. Fixed.